src/components/primitives/ProfileAvatar/ProfileAvatar.tsx

```diff
@@ -1,3 +1,30 @@
-import ContrastBox from "../../../styleguide/ContrastBox/ContrastBox";
+import * as React from "react";
 
-export default ContrastBox;
+import styled, { withTheme } from "styled-components";
+import {ProfileAvatarTypesStr} from "../../../types/ProfileAvatarTypes";
+import {setProfileAvatarBySize} from "../../../utils/components/primitives/profileAvatarUtils";
+
+export interface ProfileAvatarProps
+  extends React.ImgHTMLAttributes<HTMLImageElement> {
+  style?: React.CSSProperties;
```

Comment:
style is already in React.ImgHTMLAttributes<HTMLImageElement> 

https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.htmlimageelement.html#style